internal/driver/glfw/window.go

```diff
@@ -944,6 +949,34 @@ func (w *window) runOnMainWhenCreated(fn func()) {
 }
 
 func (d *gLDriver) CreateWindow(title string) fyne.Window {
+	var root fyne.Window
+	d.windowLock.RLock()
+	count := 0
+	for _, w := range d.windows {
+		if w.(*window).visible {
+			count++
+			if count == 1 {
+				root = w
```

Comment:
I think you might get away with a `break` here and a boolean for if there are any visible windows?

Reply:
Good hint thanks